sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -22,7 +22,7 @@ import org.apache.spark.sql.Column
 import org.apache.spark.sql.catalyst.expressions._
 
 /**
- * Utility functions for defining window in DataFrames.
+ * Utility functions for defining window in Datasets.
```

Comment:
These are used with DataFrames, right? At least that's what I have used Window for, am I missing something?